examples/CMakeLists.txt

```diff
@@ -46,17 +46,39 @@ IF(DEAL_II_COMPONENT_EXAMPLES)
     PATTERN "DTMB-5415_bulbous_bow.iges"  # step-54
     )
 
-  MESSAGE(STATUS "Setting up examples - Done")
-ENDIF()
+  #
+  # Set up all executables:
+  #
+  FILE(GLOB _steps ${CMAKE_CURRENT_SOURCE_DIR}/step-*/*.cc)
+  FOREACH(_step ${_steps})
+    GET_FILENAME_COMPONENT(_name ${_step} NAME_WE)
+    GET_FILENAME_COMPONENT(_directory ${_step} DIRECTORY)
+    SET(_set_up TRUE)
+    IF(EXISTS "${_directory}/dependency")
+      FILE(READ "${_directory}/dependency" _dependency)
```

Comment:
Maybe, but it also duplicates information. That's almost never successful.

I'm also a fan of clean directories. If we can hide something in a subdirectory that the normal user does not need to see every day, then let's do that.